src/main/java/com/codejoust/main/dto/game/GameMapper.java

```diff
@@ -84,9 +85,22 @@ public static SubmissionDto submissionToDto(Submission submission) {
             return null;
         }
 
+        List<SubmissionResultDto> notHidden = new LinkedList<>();
+
+        if (submission.getResults() != null) {
+            for (int i = submission.getResults().size() - 1; i >= 0; i--) {
+                if (!submission.getResults().get(i).isHidden()) {
+                    notHidden.add(mapper.map(submission.getResults().get(i), SubmissionResultDto.class));
+                }
+            }
+        }
+
+
         mapper.getConfiguration().setMatchingStrategy(MatchingStrategies.LOOSE);
+        SubmissionDto returned = mapper.map(submission, SubmissionDto.class);
```

Comment:
nit: Change to `submissionDto`, or something that better describes what this object is. We can tell that it is returned, I'd rather have information that says what is actually being returned in the first place.